### DISCUSSION TITLE
vod upload via API: notMp4File

### DISCUSSION THREAD

**Discussion by tbobker123** [ant-media/Ant-Media-Server#6657] [General]
Good morning Antmedia team! 

I am trying to upload a mp4 file using the api but experiencing this error in the screenshots:

<img width="1483" alt="Screenshot 2024-09-18 at 09 52 51" src="https://github.com/user-attachments/assets/9ad76bab-7d12-4d77-8af8-29115a2265ec">

<img width="1581" alt="Screenshot 2024-09-18 at 09 52 46" src="https://github.com/user-attachments/assets/df56ba67-f71d-4721-b500-4dd4ed703be3">

Tried this curl example:

```
curl -X POST 'http://localhost:5080/testapp/rest/v2/vods/create?name=filename' \
  -H 'Content-Type: multipart/form-data' \
  -H 'Authorization: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.QHhB2VZt4G0vxZshTwDWzJjBY103vWNDFudV3HYbKcw' \
  -H 'Content-Type: multipart/form-data' \
  -F 'file=@/Users/tbobker/Downloads/sample-mp4-files-sample_640x360.mp4'
```

As far as I can see, this is a MP4 file. 

<img width="1183" alt="Screenshot 2024-09-18 at 09 56 08" src="https://github.com/user-attachments/assets/78411c64-7208-45e6-9871-b895b91f0bca">

Can you please help me understand what am I doing wrong?

Version: Community Edition 2.10.0 20240705_1023

**Comment by filip-ams** [comment]
Hi @tbobker123,

I hope you are doing well.

It looks like you didn't provide a file path here:

![image](https://github.com/user-attachments/assets/93ef9f57-b515-4aa2-a016-0da6f9429419)

Also a quick tip to avoid these issues when calling our REST API is to execute the command with your parameters on our website, like so:

![image](https://github.com/user-attachments/assets/08a5eb3b-06ae-4388-9617-afa6a3a7030b)

I hope this helps and If you feel satisfied with Ant Media feel free to leave a review on [Capterra](https://www.capterra.com/p/183566/Ant-Media-Server/)

**Reply by tbobker123** [reply]
Thank you @filip-ams that worked!

**Reply by filip-ams** [reply]
Anytime !
